pkg/agent/storage/utils_test.go

```diff
@@ -48,3 +50,63 @@ func TestCreate(t *testing.T) {
 	expectedMode := os.FileMode(0777)
 	g.Expect(mode.Perm()).To(gomega.Equal(expectedMode))
 }
+
+func TestFileExists(t *testing.T) {
```

Comment:
what are these tests for? seems like not testing kserve code

Reply:
These were testing agent storage utils.go